### DISCUSSION TITLE
DDRMenu: how to check Node hidden/visible

### DISCUSSION THREAD

**Discussion by thienvc** [dnnsoftware/Dnn.Platform#4889] [Q&A]
When i includeHidden=true, how to check node is hidden in Razor?

**Comment by bdukes** [comment]
You have to use the `TabId` property to get the `TabInfo` from [`TabController.Instance.GetTab(node.TabId, Null.NullInteger)`](https://docs.dnncommunity.org/api/DotNetNuke.Entities.Tabs.ITabController.html#DotNetNuke_Entities_Tabs_ITabController_GetTab_System_Int32_System_Int32_) and then you can check [`TabInfo.IsVisible`](https://docs.dnncommunity.org/api/DotNetNuke.Entities.Tabs.TabInfo.html#DotNetNuke_Entities_Tabs_TabInfo_IsVisible)

**Reply by thienvc** [reply]
Thank you
I think GetTabInfo in ForEach for Menu I think will reduce performance
I will find another way
